File: Backend/Electronics/Models/Product.cs
using System;

namespace Electronics.Models
{
    public class Product
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Brand { get; set; }
        public string Color { get; set; }
        public string Dimentions { get; set; }
        public double Weight { get; set; }
        public string Os { get; set; }
        public double Discount { get; set; }
        public byte[] ImageData { get; set; }
        public string ImagePath { get; set; }
        public int Price { get; set; }
    }
}

File: Frontend/front/src/app/models/product.ts
import { Guid } from "guid-typescript";

export class Product {
    id: Guid
    name: string;
    brand: string;
    category: string;
    color: string;
    dimention: string;
    weight: any;
    os: string;
    discount?: any;
    imageData?: string;
    imagePath?: string;
}
